Plots/Panels/NCL_panel_20.py

```diff
@@ -0,0 +1,232 @@
+                                     ylim=(-90, 90),
+                                     xticks=np.arange(-180, 181, 30),
+                                     yticks=np.arange(-90, 91, 30))
+
+    # Use geocat.viz.util convenience function to add minor and major ticks
+    gvutil.add_major_minor_ticks(ax, labelsize=8)
+
+    # Use geocat.viz.util convenience function to make plots look like NCL
+    # plots by using latitude, longitude tick labels
+    gvutil.add_lat_lon_ticklabels(ax)
+
+    # Remove the degree symbol from tick labels
+    ax.yaxis.set_major_formatter(LatitudeFormatter(degree_symbol=''))
+    ax.xaxis.set_major_formatter(LongitudeFormatter(degree_symbol=''))
+
+    # Use geocat.viz.util convenience function to set titles and labels
+    gvutil.set_titles_and_labels(ax,
+                                 maintitle='300mb',
+                                 maintitlefontsize=8)
+
+
+###############################################################################
+# Plot:
+
+fig = plt.figure(figsize=(12, 10))
+
+# Create grid with two rows and two columns
+# Use `height_ratios` to adjust the relative heights of the rows
+grid = gridspec.GridSpec(nrows=2,
+                         ncols=2,
+                         height_ratios=[0.55, 0.45],
+                         hspace=0.1,
+                         figure=fig)
+
+# Choose the map projection
+proj = ccrs.PlateCarree()
+
+# Add the subplots
+ax1 = fig.add_subplot(grid[0])  # upper left cell of grid
+ax2 = fig.add_subplot(grid[1])  # upper right cell of grid
+ax3 = fig.add_subplot(grid[2], projection=proj)  # lower left cell of grid
+ax4 = fig.add_subplot(grid[3], projection=proj)  # lower right cell of grid
+
+# Draw coastlines on maps
+ax3.coastlines(linewidth=0.5)
+ax4.coastlines(linewidth=0.5)
+
+# Use geocat.viz.util convenience function to set titles without calling
+# several matplotlib functions
+gvutil.set_titles_and_labels(ax1,
+                             maintitle='Time=0',
+                             maintitlefontsize=14,
+                             ylabel=U_0.long_name,
+                             labelfontsize=14)
+gvutil.set_titles_and_labels(ax2,
+                             maintitle='Time=1',
+                             maintitlefontsize=14)
+
+# Draw tick labels on the right side of the top right plot
+ax2.yaxis.tick_right()
+
+# Use helper function to create linegraphs with identical axes parameters
+format_linegraph_axes(ax1)
+format_linegraph_axes(ax2)
+
+# Remove tick labels for top left plot
+ax1.set_yticklabels([])
+
+# Use helper function to create contour plots with identical axes parameters
+format_contour_axes(ax3)
+format_contour_axes(ax4)
+
+# Add left and right titles for both map plots
+ax3.set_title(U_0.long_name, loc='left', y=1.04, fontsize=8)
+ax3.set_title('Time = 0', loc='right', y=1.04, fontsize=8)
+ax4.set_title(U_0.long_name, loc='left', y=1.04, fontsize=8)
+ax4.set_title('Time = 0', loc='right', y=1.04, fontsize=8)
```

Comment:
```suggestion
ax4.set_title('Time = 1', loc='right', y=1.04, fontsize=8)
```
For consistency with NCL plot